### DISCUSSION TITLE
wire:naviage links break in flickity carousel

### DISCUSSION THREAD

**Discussion by patriktoth67** [livewire/livewire#6721] [7. Bugs (archived)]
### Livewire version

3.0.2

### Which PHP version are you using?

PHP 8.1

### Steps To Reproduce

I'm trying to make an SPA with livewire, so I use full page components as route parameters, and all links with wire:navigate.
When I click the browsers back or forward button, none of the livewire functions run. Not mount(), not even render(). 
In the view file hovewer, alpineJS runs all it's functions.
Since none of the php methods ran, the $wire object doesn't have anything, leaving you with a million errors.

Should I just force a refresh for browser back and forward, or is there a method to solve this?

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by jeffegiovani** [comment]
I had the same problem, so I replaced Flickity with Splide.js and implemented a function that prevents clicking when dragging slides.

```
let splide = new Splide( '.splide' {yourOptionsHere});
var splideSlidesLinks = document.querySelectorAll('a.splide__slide');

splide.mount();

splide.on('drag', function (e) {
    setTimeout(() => {
        splideSlidesLinks.forEach(slide => slide.style.pointerEvents = "none");
    }, 50);
});

splide.on('dragged', function (e) {
    splideSlidesLinks.forEach(slide => slide.style.pointerEvents = "all");
});
```

**Comment by 0smr** [comment]
I encountered this problem today with Livewire 4 and ended up creating a custom directive that calls `Livewire.navigate` only when a link is clicked.

There are also multiple issues with `wire:navigate`. Most importantly, it fetches the target page on `mousedown`, which can cause unwanted requests even when the user is not actually clicking the anchor element.

Unfortunately, this behavior is problematic inside sliders and carousels.

Is there a better solution to this issue?

Wouldn’t it be better to have a `.click` or similar modifier so that fetching and navigation occur only when there is no drag, and only after the `click` event?
